ipatests/test_ipalib/test_filestore.py

```diff
@@ -0,0 +1,985 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Module provides unit tests to verify that the FileStore code works.
+"""
+
+from __future__ import absolute_import
+from hashlib import sha256
+from ipaplatform.paths import paths
+
+try:
+    # install subpackages are not shipped with wheels
```

Comment:
You neither need ``needs_install_package`` marker nor this try/except block. Tests for any ``install`` subpackage are in the ``ipatests.test_install`` package. The directory is automatically skipped in client-only installation.

Reply:
Fixed.